### DISCUSSION TITLE
docker-entrypoint.sh error

### DISCUSSION THREAD

**Discussion by ImranJM1425** [postalserver/postal#2118] [Help with using Postal]
Hi after running postal initialize I get this error. 

"exec /docker-entrypoint.sh: exec format error"

This is on ubuntu arm server with mariadb 10.5, and the default rabbitmq. All settings are unchanged from the sample config.

**Comment by willpower232** [comment]
> ubuntu arm server

I'm reasonably sure the default images aren't ARM friendly and you'll probably need to make your own if you're going to stay on an ARM server
